scripts/lib/install

```diff
@@ -1,61 +1,81 @@
 #!/usr/bin/env bash
 set -xe
 
+# Specify options for apt
+APT_OPTIONS="${APT_OPTIONS:-}"
+# Install additional packages using apt
+ADDITIONAL_PACKAGES=${ADDITIONAL_PACKAGES:-}
+# Call the default type "voyager", for docker it's "dockervoyager"
+DEPLOYMENT_TYPE="${DEPLOYMENT_TYPE:-voyager}"
+# Can be for example "dockervoyager" or voyager
+VOYAGER_CLASS="${VOYAGER_CLASS:-voyager}"
```

Comment:
I think we should include the `zulip::` in here, and rename it to something like "PUPPET_CLASSES", because this is actually a comma-separated list, not a single value.  E.g. you might want "zulip::app_frontend, zulip::memcached"